styles.css

```diff
@@ -98,3 +145,208 @@ h1 {
+  text-align: center;
+  justify-content: space-around;
+  min-width: 45px;
+}
+
+.container-projects-element li {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  text-align: center;
+  color: #3a4a42;
+  background-color: #ebf0ee;
+  min-height: 20px;
+  min-width: 10px;
+  border-radius: 4px;
+  margin: 0 5px 0 5px;
+  padding: 6px 10px 1px 10px;
+}
+
+.container-btn {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  margin-bottom: 15px;
+  min-height: 40px;
+  width: 122px;
+  border-radius: 4px;
+  background: #36b37f;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 20px;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  width: 315px;
+}
+
+.aboutme-title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 327px;
+  text-align: center;
+  background-image: url('images/about-me-mobile.png'), url('images/about-me-mobile_1.png');
+  background-repeat: no-repeat;
+  background-position: left bottom, right top;
+}
+
+.aboutme-title h3 {
+  margin: 0 5px 20px 5px;
+  display: flex;
+  height: 48px;
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  text-align: center;
+}
+
+.for-about-button {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  background: #36b37f;
+  border-radius: 4px;
+}
+
+.about-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  height: 48px;
+  min-width: 135px;
+  border-radius: 4px;
+  background: #36b37f;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  margin: 40px auto;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.about-container {
+  background: #ebf0ee;
+  border: 1px solid #ebf0ee;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  border-radius: 8px;
+  height: 352px;
+  width: 327px;
+  padding: 25px;
+  margin-top: 15px;
+}
+
+.container-projects-skill {
+  display: flex;
+  flex-direction: row;
+  text-align: center;
+  justify-content: space-around;
+  min-width: 45px;
+}
+
+.container-projects-skill li {
+  display: flex;
+  font-family: 'Inter', sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  text-align: center;
+  color: #36b37e;
+  background-color: #fff;
+  min-height: 20px;
+  min-width: 10px;
+  border-radius: 4px;
+  margin: 0 0 0 5px;
+  padding: 6px 10px 1px 10px;
+}
+
+input[type=button]:hover,
+:active {
+  background-color: #36b37e;
+  border: 1px solid #000;
+}
```

Comment:
Please remove or fix this line of code 😊 since when a user press the button, the whole website is taking the #36b37e color. I think it's better to add a hover for each specific button.